src/Curation/ItemCuration/ItemCuration.model.ts

```diff
@@ -14,4 +15,20 @@ export class ItemCuration extends Model<ItemCurationAttributes> {
       FROM ${raw(this.tableName)} ic
       INNER JOIN ${raw(Item.tableName)} i ON i.id = ic.item_id AND i.collection_id = ${collectionId}`)
   }
+
+  static async getItemCurationCountByThirdPartyId(
+    thirdPartyId: string
+  ): Promise<{ count: number }[]> {
+    return this.query(
```

Comment:
This method is great. Two suggestions:
- In other counts in the project we usually just return the number, to avoid having to burden the caller with knowing that only the first result is relevant (like you did on `fetchMaxItemsByThirdParty`). If you think it looks better we could change it.
- You can type `query` too! so you can do `const r = this.query<{ count: number }>`, where r'd be `{ count: number }[]`